nixos/modules/config/zram.nix

```diff
@@ -26,15 +26,16 @@ in
         description = ''
           Enable in-memory compressed swap space provided by the zram kernel
           module. It is recommended to enable only for kernel 3.14 or higher.
+          See https://www.kernel.org/doc/Documentation/blockdev/zram.txt
         '';
       };
 
       numDevices = mkOption {
-        default = 4;
+        default = 1;
         type = types.int;
         description = ''
           Number of zram swap devices to create. It should be equal to the
-          number of CPU cores your system has.
+          number of CPU cores your system has for kernel < v3.15.
```

Comment:
We don't support kernels that old anymore.

Reply:
Ah, good, let me adjust the message.